src/view/form-edit-view.js

```diff
@@ -13,139 +15,292 @@ const createFormEditTemplate = (event) => {
-          ${type}
-        </label>
-       <input 
-          class="event__input  event__input--destination" 
-          id="event-destination-${id}" 
-          type="text" name="event-destination" 
-          value="${destination.name}" 
-          list="destination-list-${id}"
-       >
-       <datalist id="destination-list-${id}">
-          ${CITIES.map((city) => `<option value=${city}></option>`).join ('')}
-       </datalist>
-     </div>
-
-      <div class="event__field-group  event__field-group--time">
-        <label class="visually-hidden" for="event-start-time-${id}">From</label>
-        <input class="event__input  event__input--time" id="event-start-time-${id}" type="text" name="event-start-time" value="${formatDate(dateFrom,'DD/MM/YY HH:mm')}">
-          &mdash;
-        <label class="visually-hidden" for="event-end-time-${id}">To</label>
-        <input class="event__input  event__input--time" id="event-end-time-${id}" type="text" name="event-end-time" value="${formatDate(dateTo,'DD/MM/YY HH:mm')}">
-      </div>
-
-      <div class="event__field-group  event__field-group--price">
-        <label class="event__label" for="event-price-${id}">
-          <span class="visually-hidden">Price</span>
-            &euro;
-        </label>
-        <input class="event__input  event__input--price" id="event-price-${id}" type="text" name="event-price" value="${basePrice}">
-      </div>
-
-      <button class="event__save-btn  btn  btn--blue" type="submit">Save</button>
-      <button class="event__reset-btn" type="reset">Delete</button>
-      <button class="event__rollup-btn" type="button">
-        <span class="visually-hidden">Open event</span>
-      </button>
-     </header>
-
-     <section class="event__details">
-      <section class="event__section  event__section--offers">
-        <h3 class="event__section-title  event__section-title--offers">Offers</h3>
-      
-        <div class="event__available-offers">
-        ${offers.map(({title, price}) =>
-      `<div class="event__offer-selector">
-        <input class="event__offer-checkbox  visually-hidden" id="event-offer-${title}-${id}" type="checkbox" name="event-offer-${title}" checked>
-        <label class="event__offer-label" for="event-offer-${title}-${id}">
-          <span class="event__offer-title">Add ${title}</span>
-            &plus;&euro;&nbsp;
-          <span class="event__offer-price">${price}</span>
-        </label>
-        </div>`).join('')}
-       </div>
-      </section>
-
-      <section class="event__section  event__section--destination">
-        <h3 class="event__section-title  event__section-title--destination">Destination</h3>
-        <p class="event__destination-description">${destination.description}</p>
-      </section>
-     </section>
-    </form>
-  </li>`);
+  const allAvailableOption = eventOffers(type).offers;
+  const hideOffersContainer = () => !allAvailableOption.length ? 'visually-hidden' : '';
```

Comment:
Это условие лучше добавить прямо в шаблон. Далее при изучении реакта ты будешь часто видеть подобное использование. 
```suggestion
  Boolean(allAvailableOption.length) ? allAvailableOption.map((............) : '';
```